src/Compilers/CSharp/Portable/Binder/LocalScopeBinder.cs

```diff
@@ -268,7 +268,8 @@ protected LocalFunctionSymbol MakeLocalFunction(LocalFunctionStatementSyntax dec
             return new LocalFunctionSymbol(
                 this,
                 this.ContainingMemberOrLambda,
-                declaration);
+                declaration,
+                this.InUnsafeRegion);
```

Comment:
The same question applies to this.ContainingMemberOrLambda argument.

---
In reply to: [91799297](https://github.com/dotnet/roslyn/pull/15596#discussion_r91799297) [](ancestors = 91799297)